### ISSUE TITLE
Is CC1 to IO0 not an issue?

### ISSUE THREAD

**Issue by ayavilevich** [Xinyuan-LilyGO/T-U2T#6]
Hi,

Normally CC1 and CC2 on the device side will have 5K ohm resistors to ground. This makes sure that USB-C chargers can charge the device properly.
In the T-U2T design, the device has IO0 connected to CC1. IO0 can be in various states; high, low, etc. Doesn't this break proper USB-C charging for any "T-U2T" compatible device?

**Comment by f-hoepfinger-hr-agrartechnik** [comment]
you mean if you use a USB for Flashing and SAME USB for Charging ?

**Comment by ayavilevich** [comment]
> you mean if you use a USB for Flashing and SAME USB for Charging ?

Yes, normally a board will have one USB connector for both flashing and charging. Not necessary at the same time.
